usom/info.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "usom",
+  "version": "1.0.0",
+  "label": "USOM",
+  "description": "USOM Lookup Connector",
+  "publisher": "Community",
+  "icon_small_name": "usom_small.png",
+  "icon_large_name": "usom_large.png",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "investigation",
+  "help_online": "https://www.usom.gov.tr",
```

```suggestion
  "help_online": "",
```
We will provide he documentation link.